tests/Http/SubscriptionUpdateTest.php

```diff
@@ -54,6 +54,7 @@ public function testRemoveSubscription()
 
         $this->assertResponseStatus(200);
         $this->seeJsonField('data.email_subscription_topics', []);
+        $this->assertEquals(['news'], $user->fresh()->email_subscription_topics);
```

Comment:
Ooh tricky, looks like this test is failing. I think we'd actually want this new assertion in the 
`testAddSubscription` test above:
https://github.com/DoSomething/northstar/blob/250e7f9fd503cdb48410aa76eb32131d666d4a32/tests/Http/SubscriptionUpdateTest.php#L7-L21

Here, we're testing the _removal_ of the subscription so we'd actually expect this to _not_ be in the database. 

Though, it could be nice to even leave this in while we have it per Dave's point, and instead assert that this does _not_ show up, and that the `email_subscription_topics` list is now empty 🤔 
```suggestion
        $this->assertEquals([], $user->fresh()->email_subscription_topics);
```

Reply:
Yep – this one should assert that the array is empty `[]`. The `testAddSubscription` and `testAddExistingSubscription ` tests above should both assert that the array is `['news']`.

Reply:
aha! thanks for the clarification guys, makes a lot of sense 🙌🏽

Reply:
So if I'm clear I'm adding the same test to `TestAddSubscription` and `testAddExistingSubscription` but the `testRemoveSubscription` test should be the only one with an empty `[]` because we're expecting it to not be tracked in the db?

Reply:
Right!

- In `testAddSubscription` we're adding a `'news'` subscription to a user who has no current subscriptions, so we'd expect their user profile to have `['news']` after the request is completed.
- In `testAddExistingSubscription`, we're adding a `'news'` subscription but the user already has it (and no other subscriptions), so we'd expect their user profile to have `['news']` after the request is completed.
- Here, in `testRemoveSubscription`, we're _removing_ a `'news'` subscription from a user that already has it (and no other subscriptions), so we'd expect their user profile to have `[]` no subscriptions after the request is completed.

Reply:
Excellent! This is so clear!